File: Fall Week 3 Greedy/python/templates/C.py
def solve():
    # Read input
    n = int(input())
    programming = []
    math = []
    pe = []

    # Store indices of children based on their skill type
    t = list(map(int, input().split()))
    for i in range(n):
        if t[i] == 1:
            programming.append(i + 1)  # Indices are 1-based
        elif t[i] == 2:
            math.append(i + 1)
        else:
            pe.append(i + 1)

    # Your code here

    # Find the maximum number of teams

    # Form teams

# Main function to handle a single test case
solve()

File: Fall Week 3 Greedy/templates/C.cpp
#include <bits/stdc++.h>
using namespace std;

void solve() {
    // read the input
    int n;
    cin >> n;
    vector<int> programming, math, pe;

    // Store indices of children based on their skill type
    for (int i = 1; i <= n; i++) {
        int t;
        cin >> t;
        if (t == 1) programming.push_back(i);
        else if (t == 2) math.push_back(i);
        else pe.push_back(i);
    }

    // Your code here

    // Find the maximum number of teams

    // Form teams
}

int main() {
    solve();
}
